subgraphs/ribbon-finance/protocols/ribbon-finance/config/deployments/ribbon-finance-eth/configurations.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
I didn't know that we were only using V2 and Earn vaults, I'll make the necessary changes.

Reply:
@shashwats22 we went back and forth internally. And I apologize but we do want the V1 vaults in there. We will keep the treasury out. And it will be good to make note of this in the readme (the scope of the vaults). Thanks for the patience

Reply:
No worries, I'll add back the v1 vaults.